nvtabular/torch_dataloader.py

```diff
@@ -249,7 +308,39 @@ def gdf_col(self, gdf):
         return (batch[0], batch[1]), batch[2].long()
 
 
-class TorchTensorBatchDatasetItr(torch.utils.data.ChainDataset):
+class AsyncTensorBatchDatasetItr(torch.utils.data.IterableDataset):
+    def __init__(self, paths, **kwargs):
```

Comment:
Documentation would help a lot here, both to describe parameters + keyword args and describe usage

Also – should the keyword arg `sub_batch_size` be renamed to `batch_size`?

Reply:
changed